scripts/ci-deploy.sh

```diff
@@ -21,29 +22,48 @@ if [[ -z ${ANSIBLE_FOLDERS} ]]; then
   ANSIBLE_FOLDERS="infrastructure"
 fi
 
+if [[ -d ${cdk_app_dir} ]]; then
+  INFRASTRUCTURE_FOLDERS=${cdk_app_dir}
+elif [[ -z ${INFRASTRUCTURE_FOLDERS} ]]; then
+  INFRASTRUCTURE_FOLDERS=${ANSIBLE_FOLDERS}
+fi
+
 # shellcheck source=.
 . "$dir"/ci-include.sh
 
-pip --quiet --disable-pip-version-check --no-color install ansible boto3 requests pyyaml awscli netaddr
-
-# install the collection for CI/CD
-ansible-galaxy collection install --force git+https://github.com/ringier-data/ops-ci-aws.git,main
-
-MODULES=("${ANSIBLE_FOLDERS//,/ }")
+MODULES=("${INFRASTRUCTURE_FOLDERS//,/ }")
 
 for module in "${MODULES[@]}"; do
   echo Deploying "${module}" module...
   pushd "${module}"
 
-  if [[ -f "requirements.txt" ]]; then
-    pip install -r requirements.txt
-  fi
+  if [[ -f "./package.json" && -f "./package-lock.json" ]]; then
+    echo "CDK app detected..."
 
-  if [[ -f "playbook.yml" ]]; then
-    ansible-playbook -e env="$ENV" -v playbook.yml
+    echo "Installing dependencies..."
```

Comment:
I'd suggest taking this message out. For two reasons:
(1) we check the logs only if something went wrong. In case of the command execution failed, CodeBuild logs the command which has a non-zero return code because of the `  set -e ` . Precisely mentioning the sub-steps does not help that much.
(2) even if we want to keep this, we should log every step. e.g. what if ` npm ci `  succeeded but ` npm run cdk deploy `    `failed? The log will look like:

> Deploying cdk-app module...
> CDK app detected...
> Installing dependencies...
> ...logs from a successful npm ci run
> ERROR: the cloudformation stack has status DELETE_FAILED

In general, even more misleading.

Reply:
Good point, I'm fine with not printing the middle-steps